packages/server/src/script/loadRegistrationTesting.ts

```diff
@@ -1,3 +1,6 @@
+// To test this script for faxes run it using --faxes on the CLI, or by
```

Comment:
We don't want to send actual emails or faxes during the test.  Is there a way to prevent this or to mock what gets sent?